CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ For semantic convention package changes, see the [semconv CHANGELOG](packages/se
 
 ### :house: (Internal)
 
+* refactor(exporter-prometheus): replace `MetricAttributes` and `MetricAttributeValues` with `Attributes` and `AttributeValues` [#4993](https://github.com/open-telemetry/opentelemetry-js/pull/4993)
```

Comment:
ah, this should go into `./experimental/CHANGELOG.md` as we're just modifying a experimental package. :slightly_smiling_face:

Reply:
thanks! I've moved the entry